fider/templates/secret-jwt-token.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fider.fullname" . }}-jwt-secret
+  labels:
+    app: {{ template "fider.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  jwt-secret: {{ randAlphaNum 64 | b64enc | quote  }}
```

Comment:
This is coming from [`crypto/rand`](https://golang.org/pkg/crypto/rand/) right? Helm is dependent on Sprig for this?

Reply:
I'm fussing here to make sure we've picked a CSPRNG and not just a PRNG

Reply:
Yes helm uses sprig: https://github.com/helm/helm-www/blob/master/content/en/docs/chart_template_guide/functions_and_pipelines.md

http://masterminds.github.io/sprig/strings.html
> #### randAlphaNum, randAlpha, randNumeric, and randAscii
> These four functions generate cryptographically secure (uses crypto/rand) random strings